src/lib/visualization/lqp_visualizer.cpp

```diff
@@ -119,6 +119,9 @@ void LQPVisualizer::_build_dataflow(const std::shared_ptr<AbstractLQPNode>& from
   VizEdgeInfo info = _default_edge;
   info.label = label_stream.str();
   info.pen_width = pen_width;
+  if (to->input_count() == 2) {
+    info.arrowhead = side == InputSide::Left ? "normallbox" : "normalrdiamond";
```

Comment:
What about capped arrows? I know that I will have to lookup this port thing each and every time. :-/
<img width="128" alt="image" src="https://user-images.githubusercontent.com/1745857/64860384-4457fc00-d62d-11e9-8176-6f28d4fb641f.png">?

Reply:
My concern is that really weird plans might have the arrows come in from a different direction than below. In that case, left and right are hard to determine. However, the current layout also has the identifier on the left / right side of the arrow.

Reply:
Never mind. I'm stupid.

Reply:
Ah right. Ok ... let's see how it plays out over time.